### DISCUSSION TITLE
2024-01-31 周题库讨论

### DISCUSSION THREAD

**Discussion by xbn2002** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#48] [General]
本周BL解锁答题如期更换链接。题目内容较前一次几乎无变化，仅有个别题干表述正反的更改，无实质选项内容的更新。

现在能否通过，已经不在于选择题能获得多少分数，关键在于32分的填空题能否拿到**至少18分以上**。
受限于个人知识水平和能力，**我们提供的主观题答案不能保证能够获得较好的分数**。
真诚的希望各位在答题的时候，**若是方便的话可以顺手记录一下自己的填答，如若后期获得高分，您也可以评论分享，为后来者提供参考。**
**我们真诚感谢各位的付出。**

### 注意  ！！！
根据最新公布的规则，已经通过审核的五级以上用户参与答题，**将可能造成回收资格或账号封禁**；
且未达到五级的用户通过非「小米社区」APP 参与答题，**也将可能面临账号封禁的处罚**。
请务必通过**正常方式**进入答题 ！

**Comment by xbn2002** [comment]
> ### 申请前必读
> 
> 1. Bootloader解锁会导致设备面临各类安全风险，并对设备造成不可预估的影响，申请解锁代表您已经充分了解相关风险
> 
> 2. 解锁后请勿将设备用于非正常用途，若检测到账号异常行为，可能导致账号被封禁
> 
> 3. 本项目仅适合正在使用小米澎湃OS且设备处于上锁状态的用户报名，如果您的设备暂未升级到小米澎湃OS，您可以继续通过原有途径获取BL锁解锁权限
> 
> 4. 答题开放时间为：工作日上午10点 - 下午18点（法定节假日为答题系统维护时间，请参阅”近期答题系统维护时间”），还请合理安排答题时间。【近期答题系统维护时间：2024年1月27日-1月28日、2月3日-2月4日、2月7日-2月20日（春节期间）】
> 
> 5. 答题成绩每周三上午9:59失效，如答题通过，请务必在周三上午10点前完成申请，避免成绩失效影响申请
> 
> 6. 报名成功不代表审核通过，解锁权限获取以最终审核结果为准；通过社区申请获得解锁权限后，若您绑定设备时该设备系统为小米澎湃OS，绑定成功后解锁等待期为72小时，请您知悉

> ### 解锁权限说明
> 
> 1. 若您当前使用的系统为MIUI且设备已经解锁，如需收到小米澎湃OS的更新推送，请您将Bootloader锁重新上锁
> 
> 2. 解锁权限次数具有时效性，到期后需重新申请。2023年12月31日24:00及之前通过申请的用户解锁权限次数保留至2024年12月31日24:00，2024年1月1日00:00及之后通过申请的用户解锁权限次数保留至申请当年（自然年）12月31日24:00
> 
> 3. 每个账号每年（自然年）最多支持3台绑定设备解锁。如您是2024年1月1日0:00前通过申请的用户，自申请通过起，至2024年12月31日24:00前最多支持3台绑定设备解锁
> 
> 4. 本解锁政策仅面向在中国大陆地区的用户和中国大陆地区销售的设备，其他地区用户及设备以当地政策为准

> ### 常见问题
> 
> 1. 我的设备已经解除了BL锁，还需要重新申请吗？
> 如果您的小米账号曾经通过原官方渠道（[http://www.miui.com/unlock/index.html](http://www.miui.com/unlock/index.html)）获得设备BL锁解锁权限，且用于申请的小米账号绑定的设备已经处于解锁状态，那么在您手动操作上锁前，设备可以继续保持解锁状态，但是已解锁且搭载系统为MIUI的设备将不会收到小米澎湃OS更新推送；
> 自2023年11月8日起，如果您的设备BL锁处于上锁状态且设备搭载的系统为小米澎湃OS，那么无论您的设备是否曾经解锁，当您希望获取BL锁解锁权限时您都需要通过小米社区申请
> 
> 2. 我还可以通过其他途径解除设备BL锁吗？
> 小米社区是官方唯一解锁申请途径，为了避免您遭受损失，请勿轻信三方解锁工具或服务
> 
> 3. 解锁前我需要做哪些准备？
> - 确定手机中已经插入SIM卡
> - 关闭手机WLAN并打开手机数据连接
> - 确保您用于申请的账号已经和当前设备绑定
> - 确保您已经对手机数据进行了备份 
> - 绑定成功后，请访问小米社区PC首页-更多功能-手机解锁获取解锁工具（解锁在电脑上进行操作）

- 答题链接: https://m.beehive.miui.com/cwkjfxs5jE2Nl002D6ENGA/

> ### BL解锁申请资格答题测试
> 
> 【无法展示成绩的原因】①请务必通过“小米社区App-内测中心-BL解锁申请-去答题”进入本次答题界面并完成答题，未使用小米社区App进行答题将视为无效答题，无法展示成绩；②等级未达5段，无法展示成绩；③答题超时提交，无法展示成绩
> 【答题违规行为说明】“代答”是违规行为，为避免您的经济损失，请勿购买相关“代答服务”
> 【答题分数有效期】答题分数仅当期有效，超过当期再次申请时需要重新答题，敬请知悉。
> 【填空题人工阅卷说明】填空题采用人工阅卷方式，人工阅卷预计在【下一答题周期开始前】完成。①您提交问卷时的实时成绩仅为【选择题得分】；②人工阅卷后，如您的【选择题+填空题总分】未达到分数线时，您的成绩将仅展示【选择题得分】；③人工阅卷后，如您的【选择题+填空题总分】达到分数线时，您的成绩将展示为【选择题+填空题总分】
> 请您充分知晓解锁和刷机存在的风险，并在可承担该风险的情况下申请Bootloader解锁
> 请您准确填写相关问题，共【15】道题目，时间限时【15】分钟，本期答题（2024年1月31日上午10:00 - 2024年2月6日下午18:00）得分高于85分（不含85分）才可通过，感谢您的支持！

**Comment by xbn2002** [comment]
- 1534116 **以下关于《小米账号使用协议》和《BL解锁申请资格答题测试》的相关要求，说法正确的有** 【多选题】
  - 5031232 如果小米发现您并非该账号初始申请注册人，小米有权在未经通知的情况下终止您使用该账号
    - ✔️ 选中。根据《小米账号使用协议》，任何情况下，您都无权使用由他人注册的账号。
  - 5031233 当您获得BL解锁资格后，通过其他途径参与本答题测试，会根据行为严重程度，面临取消资格、账号终止使用等处罚
    - ✔️ 选中。根据《BL解锁申请资格答题测试》的相关要求，您只能通过「小米社区」APP 参与答题。
  - 5031234 当您小米社区等级未达到5段及以上时，通过其他途径参与本答题测试，会根据行为严重程度，面临账号终止使用等处罚
    - ✔️ 选中。本选项的表述是对上一选项的补充。
- 1534117 **已满足BL解锁申请条件，但申请后被拒绝的原因，下列说法正确的有** 【多选题】
  - 5029895 在社交媒介恶意诋毁、谩骂、造谣等
    - ✔️ 选中。您不得在社交媒介对小米恶意诋毁、谩骂、造谣等。
  - 5029896 出现被判定为作弊的行为
    - ✔️ 选中。根据解锁申请详细说明，“代答”等账号异常行为属于违规，解锁申请不予通过，且账号将被封禁。
  - 5029897 不填写申请理由、或使用AI生成申请理由、或申请理由出现过辱骂威胁等内容
    - ✔️ 选中。您必须在申请时认真填写真实的申请理由。
  - 5029898 其他违反小米账号使用协议的相关情形
    - ✔️ 选中。您不得违反小米账号使用协议。
  - 5029899 确认无任何违规情形，被拒绝后可尝试再次申请
    - ✔️ 选中。如您无上述违规情形，在申请被拒绝后，可在 7 天后重新答题并申请。
- 1534118 **BL解锁面向发烧友和开发者开放，因此在资格筛选时，会考察是否符合发烧友的特质。下面关于发烧友的特质，说法正确的有** 【多选题】
  - 5029900 发烧友指的是对某一领域极度热爱，并对此领域的产品进行深入研究
    - ✔️ 选中。公告类型题，对上一题的正确选项做出了解释。
  - 5029901 发烧友具备探究精神，因此在遇到不了解的问题时，会在合理范围内主动探寻答案，但不会寻求不正当“捷径”或采用违规方式
    - ✔️ 选中。
  - 5029902 发烧友具备坚韧不拔的品质，因此在遇到失败或挫折时，会不断尝试和努力，但不会去造谣、诋毁
    - ✔️ 选中。
  - 5029903 发烧友具备学习精神，因此在遇到新鲜事物或新知识时，会乐于学习和钻研，但会肆意谩骂
    - ❌ 不选。发烧友是高素质人群，不会骂人。XD
  - 5029904 发烧友具备分享精神，因此在有一些好的知识或经验时，会积极分享和交流，不会去利用“资源”去“谋利”
    - ✔️ 选中。
- 1534119 **通过fastboot -r ANTI命令可以输出ANTI版本号** 【单选题】
  - 5029905 错误
    - ✔️ 选中。fastboot 不存在 -r 命令开关。正确的输出小米防回滚机制（Xiaomi Anti-Rollback）版本号命令为 fastboot getvar anti。
  - 5029906 正确
    - ❌ 不选。
- 1534120 **通过流的read()方法读取数据，返回1表示流结尾** 【单选题】
  - 5029907 正确
    - ❌ 不选。
  - 5029908 错误
    - ✔️ 选中。读取流的函数往往用返回值 -1 来表示 EOF（End of File），而不是 1。
- 1534121 **以下说法正确的有** 【多选题】
  - 5029909 在多线程开发中，对多个同步代码的同步锁，按照以一定的顺序依次获取，可以有效避免死锁发生
    - ✔️ 选中。按照一定的顺序获取多个锁的预防策略通常被称为锁的顺序规定（Lock Ordering），这意味着在整个程序中规定了一定的锁获取顺序，能够有效避免死锁发生。
  - 5029910 对顺序存储的线性表，设其长度为n，在任何位置上插入或删除操作都是等概率的。插入一个元素时平均要移动表中的(n-1)/2个元素
    - ❌ 不选。从头到尾，一共有 n+1 个可插入的位置，需要移动的元素个数分别为 n, n-1, n-2, ..., 2, 1, 0；故平均概率为: $$(n + n-1 + n-2 + ... + 2 + 1 + 0) / (n+1) = (n + 0) * (n + 1) / 2 / (n + 1) = n/2$$
  - 5029911 Xiaomi/Redmi手机，一般长按电源键即可进入Recovery
    - ❌ 不选。Xiaomi/Redmi 手机进入 Recovery 的组合键一般是「电源键」和「音量加键」，单独按「电源键」只能正常开机。
  - 5029912 使用数组实现的栈是线性存储结构
    - ✔️ 选中。数组本身就是一种线性结构。
  - 5029913 队列和栈都可以用链表来实现
    - ✔️ 选中。队列是一种先进先出（FIFO）的数据结构，用链表实现队列时，新元素被添加到链表的末尾，而删除元素则发生在链表的开头；栈是一种后进先出（LIFO）的数据结构，用链表实现栈时，新元素被添加到链表的头部，而删除元素也发生在链表的头部。
- 1534122 **请简述线刷和卡刷的步骤（8分）** （此题为人工阅卷，如出现使用AI生成、复制粘贴、雷同试卷等情况，将视为无效作答，无法通过审核） 【多项填空题】
  - 326566 线刷
    - ✔️ 至少需要输入 1 个字符。从小米官方服务器下载 TGZ 格式的全量包，将手机 BootLoader 解锁，并重启至 BootLoader 模式（手机中显示 FASTBOOT），连接电脑执行 flash_all 脚本刷入。
  - 326567 卡刷
    - ✔️ 至少需要输入 1 个字符。从小米官方服务器下载 ZIP 格式的全量包，使用「设置 - 系统更新 - 手动选择安装包」或 Recovery 模式刷入。
- 1534123 **MTK刷机的报错，原因不匹配的有** 【多选题】
  - 5029914 S_UNSUPPORTED_OPERATION：已存在同名的程序
    - ✔️ 选中。该报错意味着刷机引导文件 Scatter.txt 与设备不匹配。
  - 5029915 S_DA_EMMC_FLASH_NOT_FOUND：手机的内存己经损坏
    - ✔️ 选中。该报错意味着刷机引导文件 Scatter.txt 与设备不匹配。
  - 5029916 S_FT_NEED_DOWNLOAD_ALL_FAIL：目标ROM文件和手机系统差距过大
    - ❌ 不选。该报错意味着目标 ROM 文件和手机的系统差距过大，通常在单刷某个分区时会出现。
  - 5029917 S_BROM_CMD_STARTCMD_FAIL：没有加载好刷机包文件
    - ❌ 不选。该报错意味着没有加载好刷机包的文件或驱动没装好。
  - 5029918 S_CHIP_TYPE_NOT_MATCH：文件类型不匹配
    - ✔️ 选中。该报错意味着刷机包适配的芯片型号与设备不匹配。
- 1534124 **请简述手机BL解锁后可能产生的风险（8分）** 【填空题】
  - 0 此题为人工阅卷，如出现使用AI生成、复制粘贴、雷同试卷等情况，将视为无效作答，无法通过审核
    - ✔️ 至少需要输入 10 个字符。解锁后设备无任何安全保障，易受恶意软件攻击，个人隐私数据可能会因此泄露，并可能会在一定程度上导致设备的软硬件损毁。设备保修和购买的延保等权益将会丢失，若出现无论是否系人为导致的质量问题，均不能享受免费售后服务。部分对系统安全性依赖高的功能和服务将失效，TEE 将会熔断，设备中存储的数据将会丢失，并可能影响系统升级，甚至面临经济损失。若解锁后设备用于非正常用途，账号还将面临被封禁等风险。
- 1534125 **输入fastboot devices命令后，可能输出的结果是** 【单选题】
  - 5029919 $ wget https://xxx.xxx/xxx/xxx/xxx.img
    - ❌ 不选。该内容为用户手动输入的指令。
  - 5029920 MBR: verify success!
    - ❌ 不选。该内容为引导加载程序的日志。
  - 5029921 remount failed
    - ❌ 不选。该内容为挂载（mount）命令的报错。
  - 5029922 XMABC100-01 fastboot
    - ✔️ 选中。fastboot 子命令 devices 的输出格式为空（即无设备连接）或「<序列号> fastboot」（如有多个设备同时连接，则每行会显示一个设备）。
- 1534126 **以下说法正确的有** 【多选题】
  - 5029923 DES、AES、RC4都是对称加密算法
    - ✔️ 选中。该描述是正确的。
  - 5029924 认证常用的参数有口令、标识符、生物特征等，认证允许第三方参与验证过程
    - ✔️ 选中。在认证中，常用的参数包括口令（如密码）、标识符（如用户名）、生物特征（如指纹、虹膜、声纹、面容等）等。在一些场景中，可信的第三方可能也会参与认证，如 OAuth 开放授权协议。对于安全需求较高的认证环境，可能还会采取多方认证的方式。
  - 5029925 官方提供给用户的刷机工具是MiFlash
    - ✔️ 选中。除 MiFlash（及 MiFlashPro）外，其余工具均为非官方工具。
  - 5029926 密钥为256位的AES需要16个子密钥
    - ❌ 不选。对于 AES 算法：密钥为 128 位时（AES-128），生成 10 个子密钥; 密钥为 192 位时（AES-192），生成 12 个子密钥; 密钥为 256 位时（AES-256），生成 14 个子密钥。每个子密钥的长度都是 128 位。这些子密钥用于对数据进行轮函数的迭代。
  - 5029927 BL锁可以防止用户在设备上安装非官方或未授权的操作系统
    - ✔️ 选中。BootLoader 锁在设计上的用途即为此。
- 1534127 **简述以下几款第三方recovery的特点和区别（8分）** （此题为人工阅卷，如出现使用AI生成、复制粘贴、雷同试卷等情况，将视为无效作答，无法通过审核） 【多项填空题】
  - 326568 Team Win Recovery Project
    - ✔️ 至少需要输入 1 个字符。用户友好：TWRP 提供直观的触摸界面，易于使用；多功能：支持备份和恢复系统、刷机、分区管理等多种功能；大量设备支持：TWRP 由社区进行适配，广泛支持各种 Android 设备，包括主流和非主流设备；主题支持：用户可以通过安装主题来自定义 TWRP 的外观。
  - 326569 Pitch Black Recovery Project
    - ✔️ 至少需要输入 1 个字符。轻量级设计：PBRP 致力于保持简洁，运行速度较快；高度可定制：提供了丰富的定制选项，用户可以调整各种设置以满足个人需求；与 Substratum 主题兼容：PBRP 与 Substratum 主题引擎兼容，允许用户定制和美化界面。
  - 326570 OrangeFox Recovery
    - ✔️ 至少需要输入 1 个字符。R10 引擎：OrangeFox Recovery 使用 R10 引擎，提供高级的文件管理和操作功能；增强的备份选项：支持增量备份和快速备份，提高备份效率；Magisk 和 SuperSU 支持：内置了安装包，允许用户轻松安装 Magisk 和 SuperSU 等 root 解决方案；功能丰富的设置：提供了许多高级设置和选项，适用于对定制化有更高要求的用户。
- 1534128 **以下说法不正确的有** 【多选题】
  - 5029928 端口扫描是通过与目标系统的TCP/IP端口连接，并查看该系统处于监听或运行状态的服务
    - ❌ 不选。该描述是正确的。
  - 5029929 网络的拓扑结构是逻辑结构，与实际设备的物理连接没有必然的关系
    - ❌ 不选。该描述是正确的。
  - 5029930 9008模式下的刷机不如fastboot模式下的刷机更加彻底
    - ✔️ 选中。Qualcomm Emergency Download（EDL 或 9008）模式刷机更为底层，能够写入一些在 fastboot 模式下无法写入的分区或数据。
  - 5029931 Blowfish是一个128位分组及可变密钥长度的分组密码算法
    - ✔️ 选中。Blowfish 的分组位数为 64 位。
  - 5029932 OpenVAS是一个开放式漏洞评估系统，也可以说它是一个包含着相关工具的网络扫描器，主要用来检测远程系统和应用程序中的安全漏洞
    - ❌ 不选。该描述是正确的。
- 1534129 **以下说法正确的有** 【多选题】
  - 5029933 若计算机中的地址总线的宽度为24位，则最多允许直接访问主存储器的8MB物理空间
    - ❌ 不选。如果地址总线的宽度为 24 位，那么它可以产生 2 ^ 24 个不同的地址。每个地址对应内存中的 1 个字节，所以最多允许直接寻址的主存储器物理空间大小为 2 ^ 24 = 16777216 B，即 16 MB。
  - 5029934 RAM存储器在掉电状态下，不保留其内容
    - ✔️ 选中。RAM 掉电内容会丢失，该描述是正确的。
  - 5029935 输入“adb logcat”后，出现“waiting for device”信息，可能是由于设备没有连接计算机
    - ✔️ 选中。该信息的含义为 【等待 device 模式的设备连接】，可能出现的原因有以下几点：1）驱动未安装或出现异常，计算机无法识别设备；2）设备没有物理连接到计算机，可检查接口是否松动或数据线是否正常；3）设备不在 device 模式，或未授权计算机；4）设备没有启用「USB 调试」功能。
  - 5029936 HAProxy配置中包含的组件有frontend、daemon、listen、global等
    - ❌ 不选。daemon 不是 HAProxy 配置中所包含的组件。HAProxy 包含 global、defaults、frontend、backend、listen 共 5 个组件。
  - 5029937 刷机过程中，刷入的系统版本与原系统差异较大时，通常建议执行二清
    - ❌ 不选。当刷入系统差异较大时，如果不完整清除数据，可能会无法开机，所以二清是不够的，往往需要四清。二清指的是用户数据（Userdata 或 Data）和系统缓存（Cache）；四清指的是用户数据（Userdata 或 Data）、加密元数据（Metadata）、系统缓存（Cache）以及账户锁（Frp）。
- 1534130 **请截图本页面，并上传截图** 【文件题】
  - 0 未正确上传该页面截图或伪造截图，将视为无效答卷，无法通过审核
    - ✔️ 至少需要上传 1 个文件。请使用「小米社区」APP 答题并截图上传。
- 1534131 **以下说法不正确的有** 【多选题】
  - 5029938 线刷可以用来降级回到之前的系统版本、彻底删除设备数据等
    - ❌ 不选。该描述是正确的。
  - 5029939 软件测试最基础性的测试是单元测试
    - ❌ 不选。该描述是正确的。
  - 5029940 衡量模块的独立性的两个标准是内聚和外联
    - ✔️ 选中。在软件设计中，追求高内聚和低耦合是一个重要的原则。内聚度（Cohesion）用于度量一个模块内部各个元素（如函数、方法、类等）彼此之间的关联程度；耦合度（Coupling）用于度量一个模块与其他模块之间的关联程度。
  - 5029941 UML不属于面向对象开发方法
    - ❌ 不选。UML（Unified Modeling Language）是一种通用的建模语言，它被广泛应用于面向对象开发方法中，其本身并不是一种开发方法。
  - 5029942 构造函数的返回类型只能是void型
    - ✔️ 选中。Java 语言中构造函数与类同名，用于新建该类的实例，不写返回类型。
- 1534132 **以下FASTBOOT基本命令，对应正确的有** 【多选题】
  - 5029943 fastboot oem device-info：输出该设备的所有信息
    - ❌ 不选。子命令 oem 为制造商自定义命令。对于 Xiaomi/Redmi 手机，device-info 命令将仅能输出如下设备状态消息：Verity mode（完整性校验状态），刷入非官方镜像需要禁用，为 false 时表示已禁用完整性校验；Device unlocked（BootLoader 解锁状态），为 true 时表示已解锁；Device critical unlocked（关键分区解锁状态），为 true 时表示 BootLoader 已完整解锁；Charger screen enabled（关机充电屏幕状态）。
  - 5029944 fastboot oem poweroff：清除手机中所有数据后关机
    - ❌ 不选。子命令 oem 为制造商自定义命令。对于 Xiaomi/Redmi 手机，poweroff 命令只会关机，不会清除数据。
  - 5029945 fastboot reboot：重启手机
    - ✔️ 选中。子命令 reboot 用于重启设备。
  - 5029946 fastboot erase system：擦除系统分区
    - ✔️ 选中。子命令 erase 用于擦除指定分区的所有数据。system 为 Android 系统分区。
  - 5029947 fastboot delete：擦除所有分区
    - ❌ 不选。fastboot 不存在 delete 子命令。同时，亦不存在能直接擦除所有分区的命令。
- 1534133 **请简述对“小米BL解锁申请”的看法（8分）** 【填空题】
  - 0 此题为人工阅卷，如出现使用AI生成、复制粘贴、雷同试卷等情况，将视为无效作答，无法通过审核
    - ✔️ 至少需要输入 10 个字符。「？」（本题自行填写，言之有理即可）
- 1534134 **如何退出FASTBOOT模式？** 【多选题】
  - 5029948 在FASTBOOT模式下长按音量键约7-15秒重启进入系统
    - ❌ 不选。长按「电源键」可以强制重启，并非「音量键」。
  - 5029949 通过“fastboot reboot”及“fastboot oem poweroff”命令重启进入系统或拔线关机
    - ✔️ 选中。子命令 oem 为制造商自定义命令。对于 Xiaomi/Redmi 手机，上述命令为正确重启或关机命令。
  - 5029950 部分机型在FASTBOOT模式下不插入数据线，5分钟到半小时内会自动重启或关机
    - ✔️ 选中。部分机型在未连接（数据线拔出）时，5 分钟到半小时内才会自动重启或关机，已连接状态（数据线插入）不会自动关机。
  - 5029951 反复插拔数据线5次
    - ❌ 不选。插拔数据线不会造成任何影响。
- 1534135 **关于本问卷中的填空题，请您务必认真阅读并了解以下注意事项** 【多选题】
  - 5029952 本问卷中的填空题共计32分，采用人工阅卷方式，人工阅卷预计在【下一周期答题开始前】完成
    - ✔️ 选中。本期共有四道填空题（含多项填空题），各为 8 分。根据答题前的页面提示，填空题为人工阅卷，预计在【下一周期答题开始前】完成。
  - 5029953 您提交问卷时，系统所显示的成绩仅为【选择题得分】
    - ✔️ 选中。根据答题前的页面提示，提交后的成绩仅为选择题得分。
  - 5029954 人工阅卷后，如您的 选择题+填空题 总分【达到】分数线，系统会显示您的【选择题和填空题的总分】
    - ✔️ 选中。根据答题前的页面提示，阅卷后总分达到分数线，将显示总分。
  - 5029955 人工阅卷后，如您的  选择题+填空题 总分【未达到】分数线，系统仅显示您的【选择题得分】
    - ✔️ 选中。根据答题前的页面提示，阅卷后总分未达到分数线，仅显示选择题得分。
  - 5029956 请您及时关注答题分数变化，如您的答题分数达到分数线，请您务必在下一答题周期开始前申请解锁资格
    - ✔️ 选中。根据解锁申请详细说明，答题分数仅当期有效，过时作废。

**Reply by YaKun9** [reply]
非常nice，填空题68分

**Reply by pierrecashon** [reply]
非常感谢，68分过了

**Reply by pierrecashon** [reply]
大佬牛逼

**Reply by xbn2002** [reply]
主要看填空，各位答题的时候也可以记录一下自己的填答，如后期获得高分，希望能够发出来协助社区。我们真诚感谢各位的付出。 @pierrecashon @YaKun9

**Reply by pierrecashon** [reply]
没问题，坐等出分

**Reply by iweee1** [reply]
大佬牛逼，68

**Reply by YaKun9** [reply]
没问题，坐等出分了 @xbn2002

**Reply by pierrecashon** [reply]
出分了吗兄弟，我看你上周就68分了，怎么这周还答题 @YaKun9 我现在还没出分

**Reply by YaKun9** [reply]
@pierrecashon  还没出

**Reply by YaKun9** [reply]
@pierrecashon 上周答的是另一个号

**Reply by Zero1116** [reply]
特地等了一天再答题，希望这次不要被清零了😭

**Reply by pierrecashon** [reply]
出分了吗各位，我的还是没动静，看来这周是没戏了，出个分有这么慢吗

**Reply by YaKun9** [reply]
@pierrecashon 下周也不会出的，出分无望了

**Reply by xiaobaiiqaq** [reply]
直接提示我37分，没到及格线

**Reply by q82415670** [reply]
![fff1e2084960e73240e3cc45bbe0cb7](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/110165439/24e9285b-2a78-4070-afdd-628c48708be1)
68答题直接68分 之后还会出分吗

**Reply by 88558639** [reply]
大佬下周2月5号，还更新吗

**Reply by congWU2024** [reply]
谢谢大佬，下个周期答题先学习

**Reply by xbn2002** [reply]
> 大佬下周2月5号，还更新吗

不更了，过年了，21号恢复更新

**Comment by Megalounge** [comment]
小心，现在非5级用户账号使用答题链接进入答题会导致账号封禁，本人未知情的情况下尝试答题导致小米社区账号封禁，切记切记

**Reply by xbn2002** [reply]
会导致进入社区什么也加载不出，提示访问受限过两天再试，但是实际上就是废了

**Reply by xbn2002** [reply]
可能是我写的提醒不够醒目，等会加强下

**Reply by Megalounge** [reply]
不不，其实是我没有看discussion导致的，我是今天才看到

**Reply by Megalounge** [reply]
我靠，这么狠

**Reply by xbn2002** [reply]
> 应该还会清算之前的，我那个4级小号是上周三答的题，今天进去社区发现也被封了。

最狠的是我被踢出所有内测了，解锁资格也没了，并且最内啥的时候我账号直接被拉入了解锁封控，miui都没办法解锁

**Reply by xbn2002** [reply]
> 我靠，这么狠

你去找哪里的客服对线的，我要去解掉我的封控

**Reply by xbn2002** [reply]
> > 最狠的是我被踢出所有内测了，解锁资格也没了，并且最内啥的时候我账号直接被拉入了解锁封控，miui都没办法解锁
> 
> 离谱，真离谱

我miui绑定直接提示“账号权限不足或账号受限”，就是寄了
而且你说我违反协议，内测包括解锁你踢了那我没话说，但是为什么miui也不让我解

**Comment by YaKun9** [comment]
> > > 最狠的是我被踢出所有内测了，解锁资格也没了，并且最内啥的时候我账号直接被拉入了解锁封控，miui都没办法解锁
> > 
> > 
> > 离谱，真离谱
> 
> 我miui绑定直接提示“账号权限不足或账号受限”，就是寄了 而且你说我违反协议，内测包括解锁你踢了那我没话说，但是为什么miui也不让我解

好家伙，这么丧心病狂

**Reply by xbn2002** [reply]
直接不用小米了，幸好还没换手机，至于手上这台10还是miui，直接换个号解就行，这个号彻底废了

**Reply by YaKun9** [reply]
难顶，希望赶紧出分，顺利解锁吧，现在解个锁也太困难了

**Reply by xbn2002** [reply]
> 难顶，希望赶紧出分，顺利解锁吧，现在解个锁也太困难了

国际版解锁资格你去试了吗

**Reply by YaKun9** [reply]
@xbn2002 试了，提示满额了，明天再试试

**Reply by pierrecashon** [reply]
我看酷安有人说不出分了，人家一直等到分数清零重新答题都不出分

**Reply by YaKun9** [reply]
@pierrecashon    人工不审核了吧，临近年底了，不干活了估计。好恶心

**Reply by xiaojzhao** [reply]
兄弟，你分数更新了嘛？

**Reply by YaKun9** [reply]
@xiaojzhao 应该不会更新了，我看到很多人两三周了也没个动静

**Reply by xiaojzhao** [reply]
官方真是不想给我们解锁，是不是逼我们去找黄牛解锁，有啥办法投诉人工审题嘛？

**Reply by YaKun9** [reply]
@xiaojzhao 没有，最终解释权在人家手中，而且填空题是主观判分，即时你投诉了，人家给你判分了，给你个低分，不也是过不去吗，没用的

**Comment by Xscut** [comment]
我答完直接显示68分不及格，不知道什么情况

**Reply by pierrecashon** [reply]
因为填空题是人工审核的，小米不审核填空题，你就只有选择题得分

**Reply by Tss1221** [reply]
做了好几遍了，选择题满分，完事填空不审核

**Comment by YaKun9** [comment]
今天应该是年前最后一天审核了吧,今天不出分，年前应该是不会再出分了吧

**Reply by YaKun9** [reply]
@Kingson-Zhang 
![16523EEC](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/131952790/bd45c28a-a60a-4485-8772-bee806353802)

**Reply by 88558639** [reply]
什么地方能投诉那帮傻x，工信部管用吗

**Reply by Zezesheng** [reply]
12315不知道有没有用，这不是快315了嘛

**Reply by Zero1116** [reply]
太逆天了，上上周分数莫名其妙被清零，上周填空题没审核😅

**Comment by Zjt662** [comment]
这周出不出题库

**Reply by xbn2002** [reply]
放假了，21号会有下一版题
